language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/ImportsAcceptor.java

```diff
@@ -116,4 +124,32 @@ private TextEdit createImportTextEdit(String pkgName) {
         Range range = new Range(new Position(endLine, endCol), new Position(endLine, endCol));
         return new TextEdit(range, editText);
     }
+
+    /**
+     * Holds module information related to a particular Ballerina module.
+     */
+    public static class ModuleIDMetaData {
```

Comment:
Is there a possibility for us to reuse `CodeActionModuleId` class?

Reply:
+1 . Done

Reply:
Please check my above comment